spec/Section 2 -- Language.md

```diff
@@ -704,13 +723,13 @@ EscapedUnicode :: /[0-9A-Fa-f]{4}/
 
 EscapedCharacter :: one of `"` \ `/` b f n r t
 
-Strings are sequences of characters wrapped in double-quotes (`"`). (ex.
-`"Hello World"`). White space and other otherwise-ignored characters are
-significant within a string value.
+Strings are sequences of zero or more source characters wrapped in double-quotes
+(`"`). (ex. `"Hello World"`).
 
-Note: Unicode characters are allowed within String value literals, however
-GraphQL source must not contain some ASCII control characters so escape
-sequences must be used to represent these characters.
+Any Unicode code point other than those explicitly excluded may appear literally
+within a String value. White-space and other characters otherwise ignored
+outside of string values are significant and included. Unicode code points may
+also be represented with escape sequences.
```

Comment:
I would recommend a new syntax to support SMP escapes. You currently support only code points up to U+FFFF via `\u EscapedUnicode`, but modern languages support SMP code points via a `\u{Hex}` syntax which supports between 4 and 6 hexadecimal digits.